examples/small-study/web-ext-config.js

```diff
@@ -10,7 +10,7 @@ const defaultConfig = {
     overwriteDest: true,
   },
   run: {
-    firefox: "firefox",
+    firefox: "nightly",
```

Comment:
I am still really not a fan of this.  Studies mostly run on release.  But I will accept it.

Reply:
Not me either, but we need to sort out one or both of these first before defaulting to branded/unbranded release again: https://github.com/mozilla/shield-studies-addon-utils/issues/241, https://github.com/mozilla/shield-studies-addon-utils/issues/242